docs/source/markdown/podman-create.1.md

```diff
@@ -1451,6 +1451,11 @@ After the container is started, the location for the pidfile can be discovered w
     $ podman inspect --format '{{ .PidFile }}' $CID
     /run/containers/storage/${storage-driver}-containers/$CID/userdata/pidfile
 
+#### **--altrootdirs**=*path*
+
+Path to a directory inside the container that should be treated as a `chroot` directory.
+Any pod-managed file that is mounted into the root directory will be mounted into that location as well.
```

Comment:
Any Podman managed file, /etc/resolv.conf, /etc/hosts, /etc/hostname ..., that is mounted into the root directory ...